backend/src/main/java/com/benine/backend/http/FocussingHandler.java

```diff
@@ -45,15 +45,18 @@ public void handle(HttpExchange exchange) throws IOException {
     FocussingCamera focusCam = (FocussingCamera) cam;
     String autoOn = parsedURI.getValue("autoFocusOn");
     String setPos = parsedURI.getValue("position");
+    String speed = parsedURI.getValue("speed");
     try {
       if (autoOn != null) {
         boolean autoOnBool = Boolean.parseBoolean(autoOn);
         focusCam.setAutoFocusOn(autoOnBool);
       }
       if (setPos != null) {
         focusCam.setFocusPos(Integer.parseInt(setPos));
+      } if(speed != null) {
+        focusCam.moveFocus((int)(50 + (4.45*Integer.parseInt(speed))));
```

Comment:
To map the slider domain (which is -10 to 10) to a domain of 0-99 which is 4.95 larger so this should actually be 4.95 I'll fix it.